src/main.rs

```diff
@@ -25,6 +27,16 @@ async fn main() -> Result<(), ()> {
         .0;
 
     let lnd_args = LndCfg::new(config.address, config.cert, config.macaroon);
+    let log_level = match config.log_level {
+        Some(level) => LevelFilter::from_str(&level).unwrap_or_else(|_| {
+            panic!(
```

Comment:
Error out vs panic?

Reply:
Good call, better to keep it consistent.